lldb/unittests/Symbol/LineTableTest.cpp

```diff
@@ -177,7 +177,7 @@ CreateFakeModule(std::vector<LineTable::Sequence> line_sequences) {
     return createStringError("No .text");
 
   auto cu_up = std::make_unique<CompileUnit>(module_sp, /*user_data=*/nullptr,
-                                             /*support_file_sp=*/nullptr,
+                                             /*support_file_nsp=*/nullptr,
```

Comment:
Unless i'm mistaken, won't this assert?

Reply:
It doesn't because we go through the move constructor which doesn't have the assert, but this still goes against the design for the NonNullSharedPointer. I didn't notice because I used `sed` to rename everything. Let me fix this.